src/Magnum/Trade/AbstractImporter.h

```diff
@@ -496,6 +496,8 @@ class MAGNUM_EXPORT AbstractImporter: public PluginManager::AbstractManagingPlug
          */
         const void* importerState() const;
 
+        void openState(const void* state);
```

Comment:
There needs to be a piece of documentation here to explain what `openState` does 😉

Reply:
Similarly to the `*.cpp` file I would put this function right next to the `openData()`/`openFile()` declaration. Then you could just copypaste the docs from `openData()` and change relevant portions (mention `Feature::OpenState` etc.). That's it. There's then a few cases where `openData()`/`doOpanData()` is mentioned above in the main class docs, would be great if you'd mention the new functions there as well, but I can do that later, too.